spec/unit/lib/cloud_controller/paging/sequel_paginator_spec.rb

```diff
@@ -204,6 +204,77 @@ module VCAP::CloudController
           expect(paginated_result.total).to be > 1
         end
       end
+
+      context 'window controll config flag' do
+        let(:dataset) { AppModel.dataset }
+        let!(:app_1) { AppModel.make(guid: '1', created_at: '2024-05-15T17:23:01Z') }
+        let!(:app_2) { AppModel.make(guid: '2', created_at: '2024-05-15T17:23:02Z') }
+        let!(:app_3) { AppModel.make(guid: '3', created_at: '2024-05-15T17:23:03Z') }
+        let!(:app_4) { AppModel.make(guid: '4', created_at: '2024-05-15T17:23:04Z') }
+
+        context 'not defined' do
+          it 'uses window function' do
+            options = { page:, per_page: }
+            pagination_options = PaginationOptions.new(options)
+
+            paginated_result = nil
+            expect do
+              paginated_result = paginator.get_page(dataset, pagination_options)
+            end.to have_queried_db_times(/select/i, 1)
```

Comment:
```suggestion
            end.to have_queried_db_times(/select/i, dataset.supports_window_functions? ? 1 : 2)
```

Reply:
The test currently fails for those databases that do not support window functions. There are other tests in this file where we also have different expectations depending on the db's capabilities.

Reply:
Fixed